### PR TITLE
use new awscnfm env vars with new structure

### PR DESCRIPTION
**Pull request by xh3b4sd**
See also https://github.com/giantswarm/awscnfm/pull/291.

### CODE REVIEW

**Comment by calvix** [comment]
I think we will hit an issue because changes for 13.0.0 are also in `awscnf` `master` branch. The important change is on `hostnetworkpod` as the number changed in 13.0.0 release

TL;DR current awscnf master won't work with 12.x.x releases

**Comment by xh3b4sd** [comment]
Oh but that is not good, isn't it? 😅 Does that mean further development on `awscnfm` is blocked?

**Comment by calvix** [comment]
> Oh but that is not good, isn't it? sweat_smile Does that mean further development on awscnfm is blocked?

I was very optimistic with thinking release 13.0.0 will be merged very soon

File: tekton/tasks/aws.yaml [hunk 1]
```diff
@@ -13,14 +13,14 @@ spec:
       description: Run the aws conformance test plan of the most basic cluster scope in order to cover cluster creation and deletion.
   steps:
     - name: run-tests
-      image: quay.io/giantswarm/awscnfm:12.7.0
+      image: quay.io/giantswarm/awscnfm:12.7.1
```
Comment (xh3b4sd) [inline tekton/tasks/aws.yaml:16]:
```suggestion
      image: quay.io/giantswarm/awscnfm:13.0.0
```

**Comment by xh3b4sd** [comment]
Fishing for LGMT @giantswarm/team-firecracker-engineers.

**Review by MarcelMue — APPROVED** [review]
Approval from me but I don't know about AWSCNFM

**Review by calvix — APPROVED** [review]

**Review by njuettner — APPROVED** [review]

**Pull request merged**